src/paymentservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/paymentservice/CrcController.java

```diff
@@ -0,0 +1,49 @@
+package ca.bc.gov.open.pssg.rsbc.dps.paymentservice;
+
+import ca.bc.gov.open.pssg.rsbc.dps.paymentservice.configuration.CrcProperties;
+import ca.bc.gov.open.pssg.rsbc.dps.paymentservice.types.BeanstreamEndpointResponse;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@EnableConfigurationProperties(CrcProperties.class)
+public class CrcController {
+
+    private static final Logger logger = LogManager.getLogger(CrcController.class);
+    private final CrcProperties crcProperties;
+
+    public CrcController(CrcProperties crcProperties) {
+        this.crcProperties = crcProperties;
+    }
+
+
+    @RequestMapping(value = "/getBeanstreamEndpoints",
+            produces = { "application/xml" },
+            method = RequestMethod.GET)
+    @ApiOperation(value = "Generates application callback endpoints for Payment Service", notes = "", response = BeanstreamEndpointResponse.class, tags={ "PaymentServices"})
```

Comment:
is the notes param required if you're leaving it blank? or is this a todo for later?

Reply:
good question... it's coming from the previous one, will look it up